packages/components/src/pagination-control/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { __, sprintf } from '@wordpress/i18n';
+import classnames from 'classnames';
+import { times } from 'lodash';
+import './style.scss';
+
+interface Props {
+	currentPage: number;
+	numberOfPages: number;
+	setCurrentPage: ( page: number ) => void;
+	classNames?: string | string[];
+}
+
+const PaginationControl: React.FunctionComponent< Props > = ( props ) => {
+	const { currentPage, numberOfPages, setCurrentPage, classNames, children } = props;
+
+	const classes = classnames( 'pagination-controls', classNames?.toString()?.split( ',' ) );
```

Comment:
I think the class name here should be `pagination-control` (singular) to match the component or file name. I think just the linter doesn't flag this when inside the call to `classnames`.

Reply:
>I think just the linter doesn't flag this when inside the call to classnames

I am not sure what you mean by this part exactly? Is there supposed to be a catch to make sure the class matches the component or did I not understand that correctly haha.